pkg/cmd/build/build.go

```diff
@@ -69,7 +69,7 @@ func Run(ctx context.Context, namespace, buildKitHost string, isOktetoCluster bo
 		log.Infof("Failed to build image: %s", err.Error())
 	}
 	if registry.IsTransientError(err) {
-		log.Yellow("Failed to push '%s' to the registry, retrying ...", tag)
+		log.Yellow("Failed to push '%s' to the registry: %s, retrying ...", tag, err.Error())
```

Comment:
Let's split it in three lines:
Failed to push '%s' to the registry
err.Error()
retrying...